File: comms/bulk-messaging/sender-pools/add-sender-to-pool/add-sender-to-pool.cs
using Twilio.Comms;
using System.Threading.Tasks;
using System.Collections.Generic;

public partial class Examples
{
    public async Task Example() {
        var client = new TwilioComms(
            accountId: "TWILIO_ACCOUNT_SID",
            authToken: "TWILIO_AUTH_TOKEN"
        );

        await client.SenderPools.AddSenderAsync(
            "comms_senderpool_01h9krwprkeee8fzqspvwy6nq8",
            new List<SenderPoolsAddSenderRequestItem>(){
                new SenderPoolsAddSenderRequestItem {
                    SenderId = "comms_sender_01h9krwprkeee8fzqspvwy6nq8"
                },
            }
        );
    }

}

File: comms/bulk-messaging/sender-pools/add-sender-to-pool/add-sender-to-pool.go
package example

import (
    context "context"

    twiliocomms "github.com/twilio/twilio-comms-go/twilio"
    client "github.com/twilio/twilio-comms-go/twilio/client"
    option "github.com/twilio/twilio-comms-go/twilio/option"
)

func do() {
    client := client.NewWithOptions(
        option.WithBasicAuth(
            "TWILIO_ACCOUNT_SID",
            "TWILIO_AUTH_TOKEN",
        ),
    )
    request := []*twiliocomms.SenderPoolsAddSenderRequestItem{
        &twiliocomms.SenderPoolsAddSenderRequestItem{
            SenderId: "comms_sender_01h9krwprkeee8fzqspvwy6nq8",
        },
    }
    client.SenderPools.AddSender(
        context.TODO(),
        "comms_senderpool_01h9krwprkeee8fzqspvwy6nq8",
        request,
    )
}
